internals/secrethub/account_init.go

```diff
@@ -289,7 +289,18 @@ func (cmd *AccountInitCommand) createAccountKey() error {
 	}
 
 	fmt.Fprint(cmd.io.Stdout(), "Finishing setup of your account...")
-	_, err = client.Accounts().Keys().Create()
+
+	credential, err := cmd.credentialStore.Get()
+	if err != nil {
+		return err
+	}
+
+	fingerprint, err := credential.Fingerprint()
```

Comment:
I am not really sure anymore. It made sense in some way to take the loading of the credential out of the client. But it could use a second look.

Reply:
Seeing as we're already passing the credential to the `Keys().Create()` function, I think it might be better to do the `credential.Fingerprint()` call inside that function? Keeps this code a bit cleaner. The only reason I can think of to keep it this way is maybe if you wouldn't call `.Fingerprint()` for every use case?

Reply:
Added `Verifier` as an argument. Fingerprint can now be retrieved by `Create()`

Reply:
We decided not to pick this up right now.